datadog_checks_base/datadog_checks/base/checks/windows/perf_counters/refresh.py

```diff
@@ -67,3 +66,7 @@ def remove_server(self, server):
 
     def log_server_count(self, server):
         self.logger.info('Refresh counter set to %d for server: %s', self.servers[server], server)
+
+    def add_interval(self, interval):
```

Comment:
should it be called set_interval instead?

Reply:
I removed this function as I was addressing a comment from @clarkb7